charts/lfx-platform/templates/authelia/httproute.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright The Linux Foundation and each contributor to LFX.
+# SPDX-License-Identifier: MIT
+---
+{{ if .Values.authelia_enabled -}}
```

Comment:
This (and other places) can be updated to `authelia.enabled` now that it is supported upstream and added in #13